app.js

```diff
@@ -1,15 +1,36 @@
+
 /*
 Page elements such as buttons and UI tiles are established here:
 */
 const btnTest = document.getElementById("btnTest");
 const searchBox = document.getElementById("searchBox");
 
+/* 
+Parameters for the API are here:
+*/
+const apiUrl = "https://rest.coinapi.io/v1/assets";
+const apiKey = "7AEDAEF9-DEC4-4CC7-B682-C7AB3A36CD93"; // Only 100 calls are allowed, so you will need to generate your own key for this.
```

Comment:
a good practice would be putting this in an environment variable. Otherwise random people / bots can find your apo key here